actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -166,15 +166,21 @@ def placeholder_required?(html_options)
 
           def add_options(option_tags, options, value = nil)
             if options[:include_blank]
-              option_tags = tag_builder.content_tag_string("option", options[:include_blank].kind_of?(String) ? options[:include_blank] : nil, value: "") + "\n" + option_tags
+              blank = options[:include_blank].is_a?(String) ?
+                tag_builder.content_tag_string("option", options[:include_blank], value: "") :
+                tag_builder.content_tag_string("option", nil, value: "", label: " ")
+
+              option_tags = blank + "\n" + option_tags
```

Comment:
I guess you didn't want to make an already long line any longer here, but I don't love the duplication this introduced. How about a compromise:

```suggestion
              include_blank = options[:include_blank].kind_of?(String) ? options[:include_blank] : nil
              option_tags = tag_builder.content_tag_string("option", include_blank, value: "", label: " ") + "\n" + option_tags
```

Reply:
I believe that would always add `label=" "` to the `<option>`, which might have accessibility side effects (not sure though).  What do you think about this instead?

```suggestion
              content = (options[:include_blank] if options[:include_blank].is_a?(String))
              label = (" " unless content)
              option_tags = tag_builder.content_tag_string("option", content, value: "", label: label) + "\n" + option_tags
```

Reply:
D'oh! That was sloppy of me. Your suggestion looks good 👍